src/content/docs/style-guide/writing-docs/article-templates/intro-branch-install-doc.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Get started with the branched install component
+metaDescription: 
+redirects: 
+---
+
+import styleGuideSimpleYamlFile from 'images/style-guide_diagram_simple-yaml-file.png'
+
+import styleGuideComplexYamlFile from 'images/style-guide_diagram_complex-yaml-file.png'
+
+Our branched install component for the docs site allows our users to select one or more options, which then displays content based on those choices. 
+
+## Overview of creation process [#overview]
+
+Before you start working with the branch install component, some things to consider: 
+
+* Ensure you definitely want to use the branched install. Consider if your problem can be solved with another solution (for example, the [tab component](/docs/style-guide/structure/intro-to-components)). 
+* Browse [the branched install docs we've made](#uses) and see what's possible. Select one you'd like to clone to use as a template. 
+* Finalize the logical flow of what your options will be and what content will be displayed. Ideally you want to be near-final in your content plans before creating a branch. For this reason, you might break the work into two or more jiras. 
+
+You can build the branched install component in many ways, and it may take you a good amount of effort to get it working, especially the more complex your structure is. But, that said, here's a high level view of what a creation process will often look like: 
+
+1. Review the branched install docs, especially our example yaml files. Get to know the structure and logic of this component. 
+2. When you're ready to start, copy the .yaml file and associated folders of an install component you want to emulate, and use those as a basis for your project.
+3. Change the top-level folder name to match your project, and change the `agentName` in the yaml file to match. The `agentName` is what tells the yaml file which folder to pull its content from. (See [our example YAML files](#yaml-examples).)
+4. To make sure things are working, do a local build. 
+5. Start adding your .mdx files and editing the .yaml file. Depending on how closely your plans match the existing logic, it may be easier to strip out most of the content from the .yaml file, and add in a little bit at a time.
```

Comment:
```suggestion
5. Start adding your .mdx files and editing the yaml file. Depending on how closely your plans match the existing logic, it may be easier to strip out most of the content from the .yaml file, and add in a little bit at a time.
```